src/rendering/custom-element-widget.js

```diff
@@ -19,33 +19,31 @@ function makeDispatchFunction(element, eventName) {
 
 function subscribeDispatchers(element) {
   let {customEvents} = element.cycleCustomElementMetadata;
-  let disposables = new Rx.CompositeDisposable();
+  let dispatcherObservables = [];
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).